src/client/components/license/licenseAttribution.js

```diff
@@ -15,16 +15,13 @@ class LicenseAttribution extends React.Component {
       case GAMEMODE_EOP:
         return (
           <div className="license-attribution">
-            The card game
+            The card game{' '}
             <a href="https://www.microsoft.com/en-us/download/details.aspx?id=20303">
-              {' '}
-              Elevation of Privilege{' '}
-            </a>
-            by
-            <a href="https://adam.shostack.org/"> Adam Shostack </a> (Microsoft)
-            is licensed under
+              Elevation of Privilege
+            </a>{' '}
```

Comment:
You forgot the `by`  ;)

Reply:
thanks for spotting this, I added it again